index.js

```diff
@@ -54,7 +54,7 @@ exports.handler = function(event, _context, callback) {
                 withoutEnlargement: true,
                 fit
             };
-            return Sharp(data.Body)
+            return Sharp(data.Body, { failOnError: false })
```

Comment:
There are no rules on how to format code, but as there are no spaces between braces and a value inside in the rest of the code, I think it is better either remove spaces here -- `{failOnError: false}`, or add to everywhere.
I would prefer the first choice since this pull is not about formatting the code.